core/io/image.cpp

```diff
@@ -3217,6 +3319,9 @@ void Image::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("decompress"), &Image::decompress);
 	ClassDB::bind_method(D_METHOD("is_compressed"), &Image::is_compressed);
 
+	ClassDB::bind_method(D_METHOD("rotate_90", "dir"), &Image::rotate_90);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("rotate_90", "direction"), &Image::rotate_90);
```

I would change this to be more descriptive.